compiler/src/jdk.tools.jaotc/src/jdk/tools/jaotc/CallInfo.java

```diff
@@ -40,7 +40,9 @@ static boolean isStaticTarget(Call call) {
 
     private static boolean isStaticOpcode(Call call) {
         int opcode = getByteCode(call) & 0xFF;
-        return opcode == Bytecodes.INVOKESTATIC || opcode == Bytecodes.INVOKEDYNAMIC || opcode == Bytecodes.INVOKEVIRTUAL /* invokehandle */;
+        return opcode == Bytecodes.INVOKESTATIC || opcode == Bytecodes.INVOKEDYNAMIC || opcode == Bytecodes.INVOKEVIRTUAL /*
```

Comment:
Looks like the formater mutilated this file. Maybe let's just move the comment to its own line above?

Reply:
yes, will do